### PR TITLE
[backports][host] fix host parameter being non-functional

### PR DESCRIPTION
**Pull request by truthbk**
Thank you @marcomusso for your work. Creating the PR with the work you described in #421  - I will review immediately.

This is a backport fix. We will not typically be doing this, but this is a pretty easy fix, for a regression we introduced so it most certainly must/will be fixed.

CI broken for the old module - might not fix it.

### CODE REVIEW

File: manifests/redhat/agent6.pp [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 
 class datadog_agent::redhat::agent6(
-  $baseurl = "https://yum.datadoghq.com/beta/${::architecture}/",
+  $baseurl = "https://yum.datadoghq.com/stable/6/x86_64/",
```
Comment (truthbk) [inline manifests/redhat/agent6.pp]: This is great, but belongs in a separate PR. I will cherry-pick it to a different branch.

**Comment by marcomusso** [comment]
You're welcome. I'll try to follow up on the 1.x branch if I have other changes to contribute. Hopefully we'll leave puppet 3 soon but the changes where so minimal it was worth adding them anyway.

**Comment by marcomusso** [comment]
Is there anything blocking this to get merged into 1.x ? Something I can do?

**Comment by albertvaka** [comment]
Closing stale PR.

**Pull request closed without merging**